Backend/Backend/Controllers/AdminPanelController.cs

```diff
@@ -10,9 +12,34 @@ namespace Backend.Controllers;
 public class AdminPanelController : ControllerBase
 {
     private readonly AdminPanelService _adminService;
-    public AdminPanelController(AdminPanelService service) {
+    private readonly IAuthService _authService;
+    public AdminPanelController(AdminPanelService service, IAuthService authService) {
         _adminService = service;
+        _authService = authService;
     }
-    
-    
+
+    [HttpGet("users")]
+    public async Task<IActionResult> GetAllUsers() {
+        try {
+            return Ok(await _adminService.GetAllUsers());
```

Comment:
You should prob add a check to see if the user is logged in.

Reply:
Something that might be interesting to do on this controller, is to add to the service a version of the authService.IdentifyUserAsync where you do the same thing as that one, but check to see if its an admin at the same time.
For Reference 
![image](https://github.com/awaazo/awaazo/assets/78050750/8d4c36c5-17aa-4ab9-a814-7e854fe39255)

Reply:
This already done with the middleware called "VerifyAdmin"